### PR TITLE
update uri mapping file to new triplet format

### PR DESCRIPTION
**Pull request by sheridancbio**

### CODE REVIEW

**Comment by sheridancbio** [comment]
Three cases need a little more attention : ET/ETC, PV/PCV, SS/SEZS

**Comment by sheridancbio** [comment]
These three cases have been corrected. There were URI assigned in versions between 2018_06 and 2018_08, but these have been rescinded and set to match the original URI. This PR can now be reviewed/merged.

File: web/src/main/resources/static/index.html [hunk 1]
```diff
@@ -103,7 +103,8 @@ <h3>Mapping Tools</h3>
             <div class="col-md-12">
               <div class="card bg-light mb-3">
                 <div class="card-body">
-                  <div id='oncotree-mapping-tool'/>
+                  <div id='oncotree-mapping-tool'>
+                  </div>
```
Comment (mandawilson) [inline web/src/main/resources/static/index.html]: Ha! @ao508 requested that I change <div></div> to <div />.  Now @sheridancbio wants to undo that. :)

File: docs/resource_uri_to_oncocode_mapping.txt [hunk 2]
```diff
@@ -1,880 +1,911 @@
...
+ONC000046	hasCode	CMML
+ONC000047	hasCode	HCL
+ONC000048	hasCode	TLGL
+ONC000048	hadCode	LGLL
+ONC000049	hasCode	PCM
+ONC000049	hadCode	MM
+ONC000050	hasCode	MDS
+ONC000051	hasCode	MPN
+ONC000052	hasCode	ET
+ONC000052	hadCoce	ETC
```
[the remark this answers is not in the document]
Comment (sheridancbio) [reply docs/resource_uri_to_oncocode_mapping.txt]: 👍

[the remark this answers is not in the document]
Comment (sheridancbio) [reply docs/resource_uri_to_oncocode_mapping.txt]: fixed. Merging

**Pull request merged**